site/en/docs/webstore/group-publishers/index.md

```diff
@@ -1,115 +1,125 @@
 ---
 layout: "layouts/doc-post.njk"
-title: Set up Group Publishing
+title: Set up group publishing
 date: 2020-06-20
 description: How to share ownership of your Chrome Web Store items with other developers.
 ---
 
-You can share ownership of your items in Google Chrome Web Store with other developers by setting up
-group publishing. With group publishing, you can add developers to a Google Group, who can then act
-on your behalf. They'll have access to all the items you own and can make any changes to them that
-you can make.
+You can share ownership of your items in Google Chrome Web Store with other developers by setting up group publishing.
+
+This guide tells you how to set up group publishing.
+
+## About group publishers
+
+Use a group publisher to establish an entity that owns Chrome Web Store items, so that publishing isn't tied to any individual developer. Consider how this differs from the more basic individual publisher role.
+
+**Individual publisher** When an individual developer acts as a publisher of an item, only that developer can upload and publish updates to the item. The following diagram describes this scenario:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/VCMoTUiD0xFT5pZVj3kO.svg", alt="ALT_TEXT_HERE", width="800", height="145" %}
+
+**Group publisher** By setting up a *group publisher*, a Google Group associates a number of individuals developers into a composite entity. Any Chrome Web Store developer who belongs to the group can publish updates to the item, as depicted in the following diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/pgzFlq6rxiKFQ5NjYlBi.svg", alt="ALT_TEXT_HERE", width="800", height="395" %}
+
+Group publishers provide a number of benefits for organizations and development teams:
+
+* It's a convenient way for teams to share the publishing capability among all their members.
+* It's easy to transfer ownership of items when a developer leaves the organization.
+* It avoids unintended deletion of items that can happen when a developer leaves the org (because their account gets deleted and that individual account was the item's publisher).
+
+You can only create a group publisher once. If you need additional group 
+publishers, you'll need a separate developer account for each one.
```

Comment:
How about changing the second sentence to "but you can be added to other developer's publisher groups".

Reply:
Done